src/late_multicellular_stage/MulticellularConvolutionDispayer.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Collections.Generic;
+using Godot;
+
+/// <summary>
+///   Creature component to display it using convolution surfaces
+/// </summary>
+public partial class MulticellularConvolutionDispayer : MeshInstance3D, IMetaballDisplayer<MulticellularMetaball>
+{
+    private const float AABBMargin = 0.1f;
+
+    private StandardMaterial3D? material;
+
+    private float? overrideColourAlpha;
+
+    public float? OverrideColourAlpha
+    {
+        get => overrideColourAlpha;
+        set
+        {
+            // Due to both being nullable, this would be a bit complicated to compare with an epsilon value
+            // ReSharper disable once CompareOfFloatsByEqualityOperator
+            if (overrideColourAlpha == value)
+                return;
+
+            overrideColourAlpha = value;
+            ApplyAlpha();
+        }
+    }
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        // This is here in case we need custom shader effects at some point
+        // material = new ShaderMaterial()
+        // {
+        //     Shader = GD.Load<Shader>("res://shaders/Metaball.shader"),
+        // },
+        material = new StandardMaterial3D
+        {
+            VertexColorUseAsAlbedo = true,
+        };
+
+        ApplyAlpha();
+
+        ExtraCullMargin = AABBMargin;
+    }
+
+    public void DisplayFromList(IReadOnlyCollection<MulticellularMetaball> layout)
```

Comment:
```suggestion
    public void DisplayFromLayout(IReadOnlyCollection<MulticellularMetaball> layout)
```

I think that kind of name would be more informative.